CHANGES.md

```diff
@@ -1,6 +1,48 @@
 Release Changes List
 ====================
 
+version 1.24.1
+==============
+
+Update to the twenty-seventh public release of Chapel, April 2021  
+(see also changes below for 1.24.0)
+
+Highlights
+----------
+* significant performance improvements for InfiniBand systems
+* improved support for computing with `enum` ranges
+* closed all known memory leaks
+
+Feature Improvements
+--------------------
+* extended `param` for-loops to support `enum` ranges
+* added support for open-interval `enum` ranges
+
+Performance Optimizations / Improvements
+----------------------------------------
+* improved NUMA affinity and startup times when using a fixed heap
+* improved our approach to polling when communicating with 'gasnet/ucx'
```

Comment:
In the 1.24.0 release notes ucx was under developer-oriented changes since it's currently experimental in gasnet and we haven't documented support in our own docs. Instead of `gasnet/ucx` it was labeled `GASNet's ucx substrate`

Reply:
I've changed this to:

```
Developer-oriented changes: Runtime improvements
------------------------------------------------
* improved our approach to polling when communicating using GASNet over `ucx`
```